src/runtime/importhook.cs

```diff
@@ -9,14 +8,45 @@ namespace Python.Runtime
     /// </summary>
     internal static class ImportHook
     {
-        private static IntPtr py_import;
         private static CLRModule root;
-        private static MethodWrapper hook;
         private static IntPtr py_clr_module;
         static BorrowedReference ClrModuleReference => new BorrowedReference(py_clr_module);
 
         private static IntPtr module_def = IntPtr.Zero;
 
+        private const string LoaderCode = @"
+import importlib.abc
+import sys
+
+class DotNetLoader(importlib.abc.Loader):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
Is this init needed? It does nothing other than tell the MRO to keep looking for init, which is what it was going to do anyway if this function didn't exist.

Reply:
(same comment in the finder below)